nix/docker/context/bin/run-node

```diff
@@ -0,0 +1,276 @@
+#!/bin/bash
+
+scriptDir=$(dirname $0)
+source $scriptDir/functions
+
+echo "Running the cardano node ..."
+
+# Shift the first option by one index
+shift
+
+# Define a few defaults
+CARDANO_CONFIG_BASE="/opt/cardano/config"
+
+if [[ -z $CARDANO_CONFIG ]]; then
+  CARDANO_CONFIG="$CARDANO_CONFIG_BASE/mainnet-config.json"
+fi
+
+if [[ -z $CARDANO_TOPOLOGY ]]; then
+  CARDANO_TOPOLOGY="$CARDANO_CONFIG_BASE/mainnet-topology.json"
+fi
+
+if [[ -z $CARDANO_DATABASE_PATH ]]; then
+  CARDANO_DATABASE_PATH="/opt/cardano/data"
+fi
+
+if [[ -z $CARDANO_SOCKET_PATH ]]; then
+  CARDANO_SOCKET_PATH="/opt/cardano/ipc/socket"
+fi
+
+if [[ -z $CARDANO_LOG_DIR ]]; then
+  CARDANO_LOG_DIR="/opt/cardano/logs"
+fi
+
+if [[ -z $CARDANO_BIND_ADDR ]]; then
+  CARDANO_BIND_ADDR="0.0.0.0"
+fi
```

Comment:
It would be nice to also add optional support for `ipv6` addresses.

Reply:
Yes, noted down as a feature request in the incubator.
https://github.com/tdiesler/nessus-cardano/issues/58